onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -189,6 +190,11 @@ class PlannerImpl {
     return use_count;
   }
 
+  OrtValueIndex& InplaceBuffer(OrtValueIndex n) {
```

Comment:
Can the function could be const (even though the OrtValueIndex reference it returns can later be used for an update)?

Reply:
I am using the InplaceBuffer() as the left-hand operand later, "InplaceBuffer(reused_for) = original;"  I dont think this can be made const.